snippets/icon-accordion.liquid

```diff
@@ -1,5 +1,14 @@
 {%- if icon != 'none' -%}
-  <svg class="icon icon-accordion color-foreground-{{ settings.accent_icons }}" aria-hidden="true" focusable="false" role="presentation" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+  <svg
+    class="icon icon-accordion color-foreground-{{ settings.accent_icons }}"
+    aria-hidden="true"
```

Comment:
`aria-hidden` is not necessary here. Using `role="presentation"` already hides the icon from screen readers.

Reply:
My understanding is that if we're using `aria-hidden="true"`, the additional `role="presentation"` wouldn't have any effect at all, so I would agree we don't need both. However I'm less sure about which one of those is more desirable for this use case. They're _technically_ a little different akaik.

That said, all our icon snippets (like icon-accordion) and many other instances of svg icons in the theme are already using both. If we wanted to update one instance, we should update all of them. And I wouldn't think that should be done as part of this commit.